cypress/e2e/FlaggedApplicationsShowPage.e2e.js

```diff
@@ -5,7 +5,17 @@ const commentInputSelector = 'input[type="text"]'
 const openRowSelector = '.rt-expander.-open'
 const statusInputSelector = 'select'
 
+
+// describe('FlaggedApplicationsIndexPage', () => {
+//   // once we have fixtures, we can intercept these API endpoints:
+//   // cy.intercept("api/v1/flagged-applications?type=pending", { fixture: "foobar" }).as("pendingFlaggedRecordsWithTitle")
+//   // cy.intercept("api/v1/flagged-applications?type=duplicated)", { fixture: "foobar" }).as("duplicatedFlaggedRecordsWithTitle")
+// })
+
 describe('FlaggedApplicationsShowPage', () => {
+  // once we have fixtures, we can intercept this API endpoint:
+  // cy.intercept("api/v1/flagged-applications/record-set/*", { fixture: "foobar" }).as("flaggedRecordSet")
+
```

Comment:
**question:** are we leaving these in here for the next ticket? do we want to commit these or pull them into the next commit if so?

Reply:
Yes, this is for the next ticket. I wrote them as a convenience for who ever picks up that ticket, but I can remove them if it is trivial for the next ticket's assignee to recreate it.

Reply:
I think this is fine as well, more of a nitpick. But I'l leave it up to you.